hadoop-hdfs-project/hadoop-hdfs/src/main/java/org/apache/hadoop/hdfs/DistributedFileSystem.java

```diff
@@ -635,9 +636,15 @@ public BlockStoragePolicy getStoragePolicy(final Path src) throws IOException {
   public void setMetaEnabled(Path src, boolean metaEnabled)
```

Comment:
Does it make sense to keep this function?

Reply:
removed. fixed all the tests that were still using it